src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -24,7 +45,7 @@ export default class ViewerWrapper extends React.Component {
     }
   }
 
-  componentDidUpdate(prevProps, _prevState) {
+  componentDidUpdate(prevProps: ViewerWrapperProps, _prevState: {}) {
```

Comment:
maybe an empty `interface ViewerWrapperState {}` would make this `_prevState` type look slightly cleaner?

Reply:
It's also technically possible to remove `_prevState` entirely, depending on how we feel about incomplete argument lists. It's not used in the body of the function.

Reply:
i kind of like the completeness, to be explicit about what is going on.

Reply:
Multiple other stateless class components use an anonymous object type in generics - e.g. `... extends React.Component<Props, {}>`. Should these components also get empty state interfaces for consistency?

Reply:
I wouldn't be opposed to that.  My general threshold would be as soon as it is needing to be used in multiple places, (ie. as soon as it showed up in componentDidUpdate in addition to the class declaration), then it makes sense to give it a name.